bedrock/base/templates/includes/protocol/footer/footer-newsletter.html

```diff
@@ -8,12 +8,12 @@
   <div class="moz24-newsletter-wrapper">
     <div class="moz24-newsletter-info">
       <img loading="lazy" class="moz24-newsletter-image" src="{{ static('img/logos/m24/symbol-white.svg') }}" alt="" width="40" height="48">
-      <p>{{ ftl('footer-refresh-get-the-best') }}</p>
+      <p>{{ ftl('footer-refresh-discover-mozilla-products', fallback='newsletter-form-we-will-only-send-v2') }}</p>
```

Comment:
fallback sourced from https://github.com/mozilla/bedrock/blob/main/l10n/en/newsletter_form.ftl

if there's a better one or we'd rather remove fallback entirely, let me know